File: WinForm/Less3_hw/CompShop/Models/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CompShop.Models
{
    public class Product
    {
        public string Name { get; set; }
        public double Price { get; set; }
        public string Description { get; set; }
        public string Characteristics { get; set; }
        public override string ToString()
        {
            return $"{Name} Description: {Description} Characteristics: {Characteristics}";
        }
    }
}

File: PHP/Lesson03_cw/Product.php
<?php

class Product
{
    private string $_name;
    private float $_price;

    public function __construct(string $_name, float $_price)
    {
        $this->_name = $_name;
        $this->_price=$_price;
    }

    public function getProduct(): string{
        return "Name: $this->_name; Price: $this->_price";
    }
}